mne/utils.py

```diff
@@ -1172,8 +1172,8 @@ def run_subprocess(command, verbose=None, *args, **kwargs):
     *args, **kwargs : arguments
         Additional arguments to pass to subprocess.Popen.
 
-    Returns
     -------
+    Returns
```

Comment:
How the hell did that show up here!?

Thanks Eric. I was combing through the parts I could *remember* changing, but couldn't find anything.